src/Cron_Command.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/**
+ * Manages cron on easyengine.
+ *
+ * @package ee-cli
+ */
+
+class Cron_Command extends EE_Command {
+
+	/**
+	 * Runs cron container if it's not running
+	 */
+	public function __construct() {
+		if ( 'running' !== EE_DOCKER::container_status( 'ee-cron-scheduler' ) ) {
+			$cron_scheduler_run_command = 'docker run --name ee-cron-scheduler --restart=always -d -v ' . EE_CONF_ROOT . '/cron:/etc/ofelia:ro -v /var/run/docker.sock:/var/run/docker.sock:ro mcuadros/ofelia:latest';
+			if ( ! EE_DOCKER::boot_container( 'ee-cron-scheduler', $cron_scheduler_run_command ) ) {
+				EE::error( "There was some error in starting ee-cron-scheduler container. Please check logs." );
+			}
+		}
+	}
+
+	/**
+	 * Adds a cron job to run a command at specific interval etc.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<site-name>]
+	 * : Name of site to run cron on.
+	 *
+	 * --command=<command>
+	 * : Command to schedule.
+	 *
+	 * --schedule=<schedule>
+	 * : Time to schedule. Format is same as linux cron.
+	 *   TODO: Add @every format
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Adds a cron job to a site
+	 *     $ ee cron add example.com --command='wp cron event run --due-now' --schedule='@every 10m'
+	 *
+	 *     # Adds a cron job to host running EasyEngine
+	 *     $ ee cron add host --command='wp cron event run --due-now' --schedule='@every 10m'
```

Comment:
`ee cron add host --command='tar -czvf my_directory.tar.gz -C /root/ee-sites/example.com .' --schedule='@every 10m'`

Example cron command for running on host.